common/main-container/index.json-ld

```diff
@@ -0,0 +1,55 @@
+{
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+	"title": {
+		"en": "Main Container",
+		"fr": "Conteneur Principal"
+	},
+	"description": {
+		"en": "Main Container with alternate tags.",
+		"fr": "Conteneur principal avec des balises alternatives."
+	},
+	"modified": "2023-07-05",
+	"componentName": "maincontainer",
+	"processing": "baseline",
+	"status": "stable",
+	"pages": {
+		"examples": [
+			{
+				"title": "Main Container",
+				"language": "en",
+				"path": "maincontainer-en.html"
+			},
+			{
+				"title": "Conteneur Principal",
+				"language": "fr",
+				"path": "maincontainer-fr.html"
+			},
+			{
+				"title": "Main Container with article",
+				"language": "en",
+				"path": "maincontainer-article-en.html"
+			},
+			{
+				"title": "Conteneur Principal avec article",
+				"language": "fr",
+				"path": "maincontainer-article-fr.html"
+			},
+			{
+				"title": "Main Container with section",
+				"language": "en",
+				"path": "maincontainer-section-en.html"
+			},
+			{
+				"title": "Conteneur Principal avec section",
+				"language": "fr",
+				"path": "maincontainer-section-fr.html"
+			}
+		]
```

Comment:
Add the report section like:

Please replace the title and the link to match the reports pre-assessement file on this PR.

```suggestion
		],
		"reports": [
			{
				"title": "Accessibility assessment #1",
				"language": "en",
				"path": "reports/a11y-2-en.html"
			},
			{
				"title": "Évaluation de l'accessibilité #1",
				"language": "fr",
				"path": "reports/a11y-2-fr.html"
			}
		]
```